main.tf

```diff
@@ -35,6 +35,8 @@ data "aws_iam_policy_document" "assume_role" {
   }
 }
 
+data "aws_partition" "current" {}
```

Comment:
this should use the count enabled logic so its disabled when enabled is false

Reply:
```data "aws_partition" "current" {
   count = local.default_service_role_count
 }
```
Ah... I understand. Sorry, done now